storage/jellyfish-merkle/src/node_type/mod.rs

```diff
@@ -16,7 +16,7 @@ mod node_type_test;
 use crate::nibble_path::NibblePath;
 use bincode::{deserialize, serialize};
 use byteorder::{BigEndian, LittleEndian, ReadBytesExt, WriteBytesExt};
-use failure::{Fail, Result, *};
+use failure::{Result, *};
```

Comment:
Isn't this redundant? * and something? Should we really be importing with * it makes it visually ambiguous where a type / function is being imported from.

Reply:
Yes it is, I must have just been moving quickly and missed that when i removed the `Fail` import.